patches/minecraft/net/minecraft/item/Item.java.patch

```diff
@@ -454,7 +454,7 @@
 +     * Queries the percentage of the 'Durability' bar that should be drawn.
 +     *
 +     * @param stack The current ItemStack
-+     * @return 1.0 for 100% 0 for 0%
```

Comment:
there's no semantic change here, the existing Javadoc is wrong. I just corrected it.